xarray/core/computation.py

```diff
@@ -1005,10 +1007,12 @@ def earth_mover_distance(first_samples,
         func = functools.partial(func, **kwargs)
 
     if vectorize:
-        if meta is None:
-            # set meta=np.ndarray by default for numpy vectorized functions
-            # work around dask bug computing meta with vectorized functions: GH5642
-            meta = np.ndarray
+        if dask == "parallelized" and meta is None:
+            # only basic checks: _apply_blockwise will raise the appropriate errors
+            if output_dtypes is not None and isinstance(output_dtypes, list):
```

Comment:
Is this ok? `_apply_blockwise` will raise informative errors if `output_dtypes` is None, if it is not a list or if it has more than one element. I don't necessarily want to repeat them here.

Reply:
fine by me.